src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = !century
+    ? people.filter((user) => user.sex === 'm')
+    : people.filter((user) => (
+      user.sex === 'm' && Math.ceil(user.died / 100) === century)
+    );
+
+  return men.reduce((mansAge, man) => (
+    mansAge + (man.died - man.born)
+  ), 0) / men.length;
 }
```

Comment:
you do the same on calculateWomenAverageAge. You can create separate function and you it here and on calculateWomenAverageAge